File: Problem sets/pset6 (Some problem sets but in python)/cash/cash.py
from cs50 import get_float

# Propmpt user for input of change owed
while True:
    change = get_float("Change owed: ")
    if change >= 0:
        break

# convert cent to dollar and round up to avoid excess values
cent = round(change * 100)

# initialize the counting of individual coins
count = 0

# subtract 25 from the change owed and continue subtracting 25 as long as its possible else move to the next denomination, count every 25 cents coin. Repeat same process for each denomination
while cent >= 25:
    cent = cent - 25
    count += 1

while cent >= 10:
    cent = cent - 10
    count += 1

while cent >= 5:
    cent = cent - 5
    count += 1

while cent >= 1:
    cent = cent - 1
    count += 1

# print the number of coins counted
print(count)

File: Problem sets/Cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("Change owed: ");
    }
    // To propmpt user for input of change owed

    while (change <= 0);
    // To ensure input is not less than 0 i.e a negative value

    int cent = round(change * 100);
    // convert cent to dollar and round up to avoid excess values

    int count = 0;
    // Declare new integer "count" to count the number of times a coin will be used to get to the change owed

    while (cent >= 25)
    {
        cent = cent - 25;
        count++;

    }
     //While loop and subtraction to divide the change by the coins (change owed - 25) then count the number of times the coin was used; repeat for the different coin denominations
    //#code snippet from @Consxious @youtube
    while (cent >= 10)
    {
        cent = cent - 10;
        count++;

    }
    //change owed - 25 - 10

    while (cent >= 5)
    {
        cent = cent - 5;
        count++;

    }
    //change owed - 25 -10 -5

    while (cent >= 1)
    {
        cent = cent - 1;
        count++;

    }
    // change owed - 25 - 10 -5 -1

    printf("%i\n", count);
    // print the number of coins used by using the count
}